src/ralph/admin/sites.py

```diff
@@ -87,5 +87,7 @@ def each_context(self, request):
         context["google_tag_manager_tag_id"] = settings.GOOGLE_TAG_MANAGER_TAG_ID
         return context
 
+    final_catch_all_view = False
```

Comment:
Without this half of ralph doesn't work
https://docs.djangoproject.com/en/5.0/releases/3.2/#django-contrib-admin